src/browser/app/App.react.js

```diff
@@ -17,14 +17,20 @@ import {connect} from 'react-redux';
 export default class App extends Component {
 
   static propTypes = {
+    actions: PropTypes.object.isRequired,
     children: PropTypes.object.isRequired,
     location: PropTypes.object.isRequired,
     msg: PropTypes.object.isRequired,
-    users: PropTypes.object.isRequired
+    persistence: PropTypes.object.isRequired
```

Comment:
@XeeD I am not passing persistenceStore, but passing persistence which is part of store which represents current state of persistence store